src/runtime/link-accounts-flow.js

```diff
@@ -184,16 +184,23 @@ export class LinkCompleteFlow {
   }
 }
 
+/**
+ * Callback for retrieving subscription request
+ *
+ * @callback saveRequestCallback
+ * @return {!Promise<!SaveSbuscriptionRequest> | !SaveSubscriptionRequest}
+ */
+
 /**
  * The flow to save subscription information.
  */
 export class LinkSaveFlow {
 
   /**
    * @param {!./deps.DepsDef} deps
-   * @param {!../api/subscriptions.SaveSubscriptionRequest} request
+   * @param {saveRequestCallback} callback
    */
-  constructor(deps, request) {
+  constructor(deps, callback) {
```

Comment:
This means that you also need to similarly modify `saveSubscription()` API in the `subscriptions.js` to switch to callback.